users/views.py

```diff
@@ -1,3 +1,43 @@
-from django.shortcuts import render
+import json, bcrypt
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from users.models     import User
+from users.validators import validate_email, validate_password
+
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            name         = data["name"],
+            email        = data['email']
+            password     = data['password']
+            phone_number = data['phone_number']
+            birthdate    = data["birthdate"]
+
+            hashed_password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
+
+            if not validate_email(email):
+                return JsonResponse({'Message' : 'Invalid Email!'}, status = 400)
```

Comment:
```suggestion
                return JsonResponse({'message' : 'Invalid Email!'}, status = 400)
```
일반적으로 소문자 구성을 많이 사용합니다.
나머지 부분들도 수정해 주세요.

Reply:
넵! 수정 완료했습니다